<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkGui</class>
 <widget class="QWidget" name="NetworkGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MuehleGui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Verbindungsaufbau</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="textName"/>
     </item>
     <item row="15" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxClient">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Client</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port:         </string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="4">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="2">
      <widget class="QCheckBox" name="checkBoxServer">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QPushButton" name="buttonClient">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Client</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QPushButton" name="buttonServer">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2" colspan="2">
      <widget class="QPushButton" name="buttonConnectServer">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Connect as Server</string>
       </property>
      </widget>
     </item>
     <item row="26" column="0" colspan="4">
      <widget class="QTextEdit" name="textBrowser">
       <property name="maximumSize">
        <size>
         <width>77215</width>
         <height>77217</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Adresse:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="16" column="0" colspan="4">
      <widget class="QCheckBox" name="checkBoxZufall">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Zufall</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="4">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wer soll beginnen?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wollen Sie Client oder Server sein?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="QPushButton" name="buttonConnectClient">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connect as Client</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0" colspan="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Historie:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Geben Sie Spielername ein:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QLineEdit" name="textPort"/>
     </item>
     <item row="7" column="0" colspan="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="buttonReset">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="textAdresse"/>
     </item>
     <item row="21" column="0" colspan="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="4">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
